### PR TITLE
clarify Duration minutes indefinite is 0

### PR DESCRIPTION
**Pull request by qrkourier**

### CODE REVIEW

File: rest_model/external_jwt_signer_create.go [hunk 1]
```diff
@@ -201,6 +201,11 @@ func (m *ExternalJWTSignerCreate) ContextValidate(ctx context.Context, formats s
 func (m *ExternalJWTSignerCreate) contextValidateTags(ctx context.Context, formats strfmt.Registry) error {
 
 	if m.Tags != nil {
+
```
[the remark this answers is not in the document]
Comment (andrewpmartinez) [reply rest_model/external_jwt_signer_create.go:204]: The version I have is: 
```
> swagger version
version: v0.29.0
commit: 09ae1192ca9a941bbb534aca09e6bdc562c95ef3
```

I see https://github.com/go-swagger/go-swagger/releases/tag/v0.31.0 this too.

[the remark this answers is not in the document]
Comment (qrkourier) [reply rest_model/external_jwt_signer_create.go:204]: :point_up_2: correct, I installed the latest stable release with `go install` from that repo.

[the remark this answers is not in the document]
Comment (andrewpmartinez) [reply rest_model/external_jwt_signer_create.go:204]: Looking through the files more, I see other changes.

Have you checked out the `ziti` repo and use `go work init; go work use .; go work use ../edge-api;` and run the edge API tests?

[the remark this answers is not in the document]
Comment (andrewpmartinez) [reply rest_model/external_jwt_signer_create.go:204]: `go test ./... -tags apitests` I believe is the command in the `ziti` repo.

**Review by andrewpmartinez — CHANGES_REQUESTED** [review]
Looking for a positive test against the `ziti` repo or generating w/ v0.29.0 (or any other version that doesn't alter every file).

**Comment by qrkourier** [comment]
> Looking for a positive test against the `ziti` repo or generating w/ v0.29.0 (or any other version that doesn't alter every file).

Tests now pass after re-generating with `swagger` v0.29.0. Spot checks of unexpected differences reveal only a change from single to multi-line comments. I'm unsure what, precisely, is responsible. I'm curious to know if re-generating on this branch with v0.29.0 on your machine reverts to single-line comments, or is the change sticky?

I'm asking because we wouldn't want dueling implementations to flap between single and multi-line in subsequent commits, but if it's sticky to multi-line, then we could leave it this way and move on.

File: scripts/generate_rest.sh [hunk 2]
```diff
@@ -1,11 +1,28 @@
-#!/bin/bash -e
+#!/usr/bin/env bash
```
Comment (andrewpmartinez) [inline scripts/generate_rest.sh:1]: Is there no update for the Powershell version?

**Review by andrewpmartinez — APPROVED** [review]

File: scripts/generate_rest.sh (hunk 2 above)
Comment (andrewpmartinez) [reply scripts/generate_rest.sh:1]: I've never done it before but: https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-linux?view=powershell-7.4

**Pull request merged**